kernel/src/vm/vmar/mod.rs

```diff
@@ -93,6 +97,38 @@ impl<R> Vmar<R> {
     }
 }
 
+#[repr(usize)]
+#[allow(non_camel_case_types)]
+#[derive(Debug, Clone, Copy, TryFromInt)]
+/// The type representing categories of Resident Set Size (RSS).
+/// See https://github.com/torvalds/linux/blob/fac04efc5c793dccbd07e2d59af9f90b7fc0dca4/include/linux/mm_types_task.h#L26..L32
+pub enum RssType {
```

Comment:
To improve readability, move all RSS related types to the end of the file. This is the newspaper style, where important and high-level stuff is put at the beginning of the page, wheres the less important and low-level details are at the end. This ordering is helpful to the readers.

Even better, you may add sub-module for RSS, where you can introduce the concept of RSS properly.